src/app/(auth)/feed/components/Card.tsx

```diff
@@ -0,0 +1,151 @@
+      }`}
+    >
+      <div className="p-5">
+        {/* Desktop Grid  - 2 Columns*/}
+        <div role="article" className="md:grid md:grid-cols-[350px_1fr] md:gap-8">
+          {/*  Right Column - Profile Info */}
+          <div>
+            <div className={`rounded-lg p-4 ${darkMode ? 'bg-capx-dark-box-bg' : 'bg-[#EFEFEF]'}`}>
+              {/* Type Badge */}
+              <div className="flex justify-start mb-4">
+                <span className={`md:text-[18px] inline-flex px-2 py-1 text-xs font-normal rounded-full border ${
+                  darkMode ? typeBadgeColorDarkMode : typeBadgeColorLightMode
+                }`}>
+                  {type}
+                </span>
+              </div>
+
+
+              {/* Profile Image */}
+              <div className="flex flex-col items-center mb-6">
+                <div className="relative w-[100px] h-[100px] md:w-[200px] md:h-[200px]">
+                  <Image
+                    priority
+                    src={avatar || noAvatarIcon}
+                    alt={pageContent["navbar-user-profile"]}
+                    fill
+                    className="object-cover rounded-[4px]"
+                    unoptimized
+                  />
+                </div>
+              </div>
+            </div>
+
+             {/* Username */}
+             <div className="mt-4 mb-6 flex items-center justify-between">
+                <h5 className={`md:text-[32px] text-xl font-bold font-[Montserrat] ${
+                  darkMode ? 'text-capx-light-bg' : 'text-capx-dark-box-bg'
+                }`}>
+                  {username}
+                </h5>
+              </div>
+          </div>
+
+          {/* Right Column - Info */}
+          <div className="mt-4 md:mt-0 flex flex-col gap-4">
+            {/* Available or Wanted Capacities */}
+            <ProfileItem
+              icon={capacitiesIcon}
+              title={capacitiesTitle}
+              items={capacities}
+              customClass={`font-[Montserrat] text-[14px] not-italic leading-[normal]`}
+            />
+
+            {/* Languages */}
+            <ProfileItem
+              icon={darkMode ? LanguageIconWhite : LanguageIcon}
+              title={pageContent["body-profile-languages-title"]}
+              items={languages}
+              customClass={`font-[Montserrat] text-[14px] not-italic leading-[normal]`}
+            />
+
+            {/* Territory */}
+            <ProfileItem
+              icon={darkMode ? TerritoryIconWhite : TerritoryIcon}
+              title={pageContent["body-profile-section-title-territory"]}
+              items={territory ? [territory] : []}
+              customClass={`font-[Montserrat] text-[14px] not-italic leading-[normal]`}
+            />
+
+            <button
+              className={`inline-flex p-2 rounded-full mr-auto ${
+                darkMode ? 'hover:bg-gray-700' : 'hover:bg-gray-100'
+              }`}
+            >
+              <Image
```

Comment:
We need to discuss a standard practice to add images to the project. The way you have done is not wrong, but it's a good practice to use standard width and height in the className, once you are using breakpoints. Along the project, I've been using images surrounded by div wrappers with position relative and fixed dimensions, and the image itself, I set the property fill. Always a good practice to check the console to see any warnings.

```suggestion
              <Image
                  src={darkMode ? AccountCircleWhite : AccountCircle}
                  alt={pageContent['body-profile-languages-title']}
                  width={27}
                  height={27}
                  className="w-[27px] h-[27px] md:w-[42px] md:h-[42px]"
                  sizes="(min-width: 768px) 42px, 27px"
                />
```